### ISSUE TITLE
Fine-tune  What the parameters look like

### ISSUE THREAD

**Issue by zl-comment** [chawins/pal#6]
If I don't change the @torch.no_grad() in the code, I can't run it and get an error that the loss forward propagation cannot be done. If I comment out @torch.no_grad(), there is not enough video memory. I would like to ask what the code looks like and what are the startup parameters for model fine-tuning?
